tools/cron/common.py

```diff
@@ -531,6 +531,7 @@ def terminate_processes(config):
 
     if 'active_hub_server_start_cmd' in config['cron']:
         terminate_process(cmdline='active_hub_server.py')
+        sleep(5)
```

Comment:
maybe in `terminate_process` we should call `process.wait()` after `process.terminate`?

Reply:
and of course we won't need to sleep here, I guess we're sleeping here for terminate to actually be done?